jupyter/jupyter-web-app/overlays/aws/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../../base
+patchesStrategicMerge:
```

Comment:
```
configMapGenerator:
- name: jupyter-web-app-config
  behavior: replace
  files:
  - spawner_ui_config.yaml -> rename `configmap_patch.yaml ` to `spawner_ui_config.yaml` here. 
```

In this case, you only need to concentrate on the configmap values. 

This is the only way to merge into same key `spawner_ui_config.yaml`, check https://github.com/kubernetes-sigs/kustomize/issues/370 for more details

Reply:
@PatrickXYS Please make the change

Reply:
Good point! Tested and it works.